OpenRA.Mods.Common/Traits/BotModules/McvManagerBotModule.cs

```diff
@@ -28,12 +28,12 @@ public class McvManagerBotModuleInfo : ConditionalTraitInfo
 		[Desc("Actor types that are able to produce MCVs.")]
 		public readonly HashSet<string> McvFactoryTypes = new();
 
-		[Desc("Try to maintain at least this many ConstructionYardTypes, build an MCV if number is below this.")]
-		public readonly int MinimumConstructionYardCount = 1;
-
```

Comment:
this will need an update rule

Reply:
your's should look similar to https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/UpdateRules/Rules/20231010/RemoveEditorSelectionLayerProperties.cs

the aforementioned class is reference in `UpdatePath.cs`